packages/doenetml-prototype/src/renderers/doenet/graph.tsx

```diff
@@ -98,9 +119,70 @@ export const Graph: BasicComponent = ({ node }) => {
         );
         setYaxis(yaxis);
 
+        board.on("boundingbox", () => {
+            if (!settingBoundingBox.current) {
+                let newBoundingBox = board.getBoundingBox();
+                let [xMin, yMax, xMax, yMin] = newBoundingBox;
+
+                // look for a change in bounding box that isn't due to round-off error
+                let xScale = Math.abs(xMax - xMin);
+                let yScale = Math.abs(yMax - yMin);
+                let diffs = newBoundingBox.map((v, i) =>
+                    Math.abs(v - (previousBoundingBox.current?.[i] ?? 0)),
+                );
+                if (
```

Comment:
Explain what this test is for.

I also prefer early return rather than nesting logic in the if statement.

Reply:
The explanation is above on line 127. Is more needed?

Reply:
I'm not entirely sure what you mean by early return. But, I changed the logic to be short-circuiting or's.